### ISSUE TITLE
创建账户时总是提示内部服务器错误

### ISSUE THREAD

**Issue by ZJM6658** [wekan/wekan-mongodb#22]
![image](https://user-images.githubusercontent.com/16357655/54589985-102ac680-4a62-11e9-88d1-49598cb55b58.png)

**Comment by xet7** [comment]
Yes, error message should be better, when email server is not configured. There should not be "Internal server error" but instead "Account created, you can login now".

是的，当未配置电子邮件服务器时，错误消息应该更好。 不应该有“内部服务器错误”，而是“创建帐户，您现在可以登录”。
